### PR TITLE
Added CCDB and CText

### PR DESCRIPTION
**Pull request by kungming2**
- [ X] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [X ] My addition is ordered alphabetically
- [X ] My submission has a useful description
- [ X] The description does not end with punctuation
- [ X] Each table column is padded with one space on either side
- [X ] I have searched the repository for any relevant issues or pull requests
- [X ] Any category I am creating has the minimum requirement of 3 items
- [X ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -338,6 +338,8 @@ API | Description | Auth | HTTPS | CORS |
 ### Dictionaries
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [Chinese Character Web API](http://ccdb.hemiola.com/) | Chinese character definitions and pronunciations | No | No | No |
```
Comment (matheusfelipeog) [inline README.md:341]: Title should not end with "... API". Every entry is an API here!

```suggestion
| [Chinese Character Web](http://ccdb.hemiola.com/) | Chinese character definitions and pronunciations | No | No | No |
```

Can you fix this, please?

File: README.md [hunk 2]
```diff
@@ -338,6 +338,8 @@ API | Description | Auth | HTTPS | CORS |
...
+| [Jisho](https://jisho.org/forum/54fefc1f6e73340b1f160000-is-there-any-kind-of-search-api) | Beta API for Jisho.org with Japanese language definitions | No | No | No |
```
Comment (matheusfelipeog) [inline README.md:342]: I didn't find the documentation for this API, the link you provided is a discussion.

If there is no documentation, this entry must be removed. Only properly documented APIs are accepted for listing here. 😅

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @kungming2 🤝 

Can you solve this review, please?

File: README.md (hunk 2 above)
Comment (matheusfelipeog) [reply README.md:342]: Here I refer to the Jisho API

File: README.md (hunk 1 above)
Comment (kungming2) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (kungming2) [reply README.md]: Got it. I swapped out its entry with the Ctext API, which is properly documented.

**Review by kungming2 — COMMENTED** [review]
Resolved

**Review by matheusfelipeog — APPROVED** [review]
Thanks for your update, looks good to me ;)

**Comment by kungming2** [comment]
@matheusfelipeog Thanks for sorting it out!

**Pull request merged**